src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -723,4 +726,35 @@ class EthService(
     }
 
   }
+
+  def getAccountRecentTransactions(request: GetAccountRecentTransactionsRequest): ServiceResponse[GetAccountRecentTransactionsResponse] = {
+    getTransactionsFromPool map { case PendingTransactionsResponse(pendingTransactions) =>
+      val pendingSent = pendingTransactions
+        .map(_.stx)
+        .filter(_.senderAddress == request.address)
+        .map(TransactionResponse(_, pending = Some(true)))
+
+      val pendingReceived = pendingTransactions
+        .map(_.stx)
+        .filter(_.tx.receivingAddress.contains(request.address))
+        .map(TransactionResponse(_, pending = Some(true)))
+
+      val numRecentBlocks = Config.Network.Rpc.txNumRecentBlocks
+
+      val bestBlockNum = appStateStorage.getBestBlockNumber()
+      val recentTxs = (0 until numRecentBlocks)
```

Comment:
Minor:

What about something like?

```scala
val start = (bestBlockNum - numRecentBlocks).max(0)
val recentTxs = (start until bestBlockNum)
     .flatMap { n => blockchain.getBlockByNumber(n) }
```

Reply:
looks good, but with `to` instead of `until` :)